packages/thirdweb/src/exports/extensions/erc7702.ts

```diff
@@ -0,0 +1,6 @@
+//ACCOUNT
+export {
+  type CreateSessionKeyOptions,
+  createSessionKey,
+  isCreateSessionKeySupported,
```

Comment:
was thinking of putting this under /in-app since its gonna be only for inapp wallets, instead of a new export